playground/samples/localization.js

```diff
@@ -0,0 +1,53 @@
+const messages = {
```

Comment:
This is fine, but you might as well just import a localize function from `ajv-i18n` rather than writing your own `convertMessagesText` function for this playground example -- I'm guessing that's simpler? I'd assume that most of the use cases from localization would use an existing function corresponding to a language from `ajv-i18n`.